src/pushsource/_impl/model/ami.py

```diff
@@ -99,6 +99,21 @@ class AmiPushItem(VMIPushItem):
     )
     """Billing codes associated with this image."""
 
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Release notes for AMI """
```

Comment:
Could these doc strings please be a little more detailed?

What are the "release notes", is it like a changelog?  Roughly how long is it?  (e.g. a single sentence, a few sentences...)

Reply:
updated this one with more detailed comment.